Assets/Scripts/SS3D/Systems/Storage/Containers/Hands.cs

```diff
@@ -114,19 +123,21 @@ public override void Update()
             {
                 return;
             }
-
-            // Hand-related buttons
-            if (Input.GetButtonDown("Swap Hands") && HandContainers.Length > 0)
```

Comment:
why remove the HandContainers > 0 check ?

Reply:
It's moved to HandleSwapHands